Paws4Me/Paws4Me/Controllers/Logic/ViewController.swift

```diff
@@ -8,109 +8,243 @@
 import UIKit
 
 class ViewController: UIViewController, UISearchBarDelegate {
+    // MARK: - IBOulets
     @IBOutlet weak private var petTable: UITableView!
     @IBOutlet weak private var searchBar: UISearchBar!
-    private var filteredPetObject: AdoptPet?
-    private var adoptPetObject: AdoptPet?
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        getAdoption()
-        petTable.delegate = self
-        petTable.dataSource = self
-        self.title = "Adoptable Animals"
-        setUpSearchbar()
-    }
+    @IBOutlet weak var viewControllerTitle: UINavigationItem!
```

Comment:
Hmmm? please look at alternatives if a ViewController is embeded in an navigation controller you have reference to it by default no need to have an IBOutlet

Reply:
Thank you. I was trying something new out to try get my back button to work but it did not work so I should have removed it.